pkg/derivation/encode.go

```diff
@@ -0,0 +1,59 @@
+package derivation
+
+import (
+	"strings"
+)
+
+func quoteString(s string) string {
```

Comment:
right now, this only encloses the string in double quotes, and doesn't do any escaping. Can you add a comment to the function header, explaining why no escaping is needed?